pretyping/cbv.ml

```diff
@@ -325,7 +375,7 @@ and cbv_stack_value info env = function
  * normalize (even under binders) the applied terms and we build the
  * final term
  *)
-let rec apply_stack info t = function
+and apply_stack info t = function
   | TOP -> t
```

Comment:
As in, left over from a merge conflict?  This PR seems to be turning a previously non-mutually recursive function into a mutually recursive one.